### PR TITLE
Kferner pull request to add README-kferner.md

### PR DESCRIPTION
**Pull request by karaferner**
pull request to add README-kferner.md file

### CODE REVIEW

File: feb-12-debugging/find_the_bug.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def sortPresHeights(presHeights):
        for j in range(0, n-i):
            # Swap if the element found is greater
            # than the next element
-           if presHeights[j][0] > presHeights[j+1][0] :
```
Comment (dailyn269) [inline feb-12-debugging/find_the_bug.py:17]: This line looks great

File: feb-12-debugging/find_the_bug.py [hunk 2]
```diff
@@ -27,33 +27,27 @@ def calc_avg_h():
    with open ('./feb-12-debugging/president_heights.csv') as csvfile:
        presidentReader = csv.DictReader(csvfile, delimiter = ",")
        for row in presidentReader:
-           presName = row['president']
```
Comment (dailyn269) [inline feb-12-debugging/find_the_bug.py:30]: Adding lines of comments here!

File: feb-12-debugging/find_the_bug.py [hunk 3]
```diff
@@ -27,33 +27,27 @@ def calc_avg_h():
...
-           numRows = numRows +1
```
Comment (dailyn269) [inline feb-12-debugging/find_the_bug.py:31]: wow! amazing

File: feb-12-debugging/find_the_bug.py (hunk 1 above)
Comment (karaferner) [reply feb-12-debugging/find_the_bug.py:17]: thanks dog

**Review by dailyn269 — COMMENTED** [review]
great stuff!

**Review by dailyn269 — APPROVED** [review]

**Review by karaferner — COMMENTED** [review]
Ready to merge

**Pull request merged**